app/queues/notifications/certificate-expiration-monitor.server.ts

```diff
@@ -0,0 +1,53 @@
+import { Queue } from 'bullmq';
+import { redis } from '~/lib/redis.server';
+import logger from '~/lib/logger.server';
+import type { DomainStatusData } from './record-cert-expiration-data.interface';
+import { prisma } from '~/db.server';
+import {
+  startCertificateExpirationMonitorWorker,
+  certExpirationQueueName,
+} from './certificate-expiration-monitor-worker.server';
+
+const { NOTIFICATIONS_REPEAT_PATTERN } = process.env;
+
+// fetch records with by status
+const getCertificatesByExpiration = async () => {
+  let monthFromToday = new Date();
+  monthFromToday.setMonth(monthFromToday.getMonth() + 1);
+  return await prisma.certificate.findMany({
+    where: {
+      AND: [
+        {
+          validTo: {
+            lte: monthFromToday,
+          },
+          timesNotified: 0,
+        },
+      ],
+    },
+    select: {
+      user: true,
+      id: true,
+    },
+  });
+};
+
+export const certificateExpirationCheckQueue = new Queue<DomainStatusData[]>(
+  certExpirationQueueName,
```

Comment:
This is another example of where co-locating code would make this easier, since I could look above this and find the name vs. having to jump between files.